diff -Nru org_tensorflow.orig/tensorflow/workspace2.bzl org_tensorflow/tensorflow/workspace2.bzl
--- org_tensorflow.orig/tensorflow/workspace2.bzl	2025-09-17 13:39:35
+++ org_tensorflow/tensorflow/workspace2.bzl	2025-09-17 13:40:35
@@ -29,6 +29,7 @@
 load("//third_party/FP16:workspace.bzl", FP16 = "repo")
 load("//third_party/gemmlowp:workspace.bzl", gemmlowp = "repo")
 load("//third_party/git:git_configure.bzl", "git_configure")
+load("//third_party/gpus:cuda_configure.bzl", "cuda_configure")
 load("//third_party/gpus:rocm_configure.bzl", "rocm_configure")
 load("//third_party/gpus:sycl_configure.bzl", "sycl_configure")
 load("//third_party/hexagon:workspace.bzl", hexagon_nn = "repo")
@@ -104,6 +105,7 @@
     # Note that we check the minimum bazel version in WORKSPACE.
     clang6_configure(name = "local_config_clang6")
     cc_download_clang_toolchain(name = "local_config_download_clang")
+    cuda_configure(name = "local_config_cuda")
     tensorrt_configure(name = "local_config_tensorrt")
     git_configure(name = "local_config_git")
     syslibs_configure(name = "local_config_syslibs")
